datagen/generate.py

```diff
@@ -205,8 +205,28 @@ def get_merged_ridership_time_series(
     return merged_time_series
 
 
+def get_ridership_percentage_array(total_ridership_time_series):
+    ridership_percentage = 100 * round(total_ridership_time_series[-1]/total_ridership_time_series[0], 2)
+    ridership_percent_array = [100 - ridership_percentage, ridership_percentage]
+    return ridership_percent_array
+
+
+def get_service_percentage_array(total_service_time_series):
+    service_percentage = 100 * round(total_service_time_series[-1]/total_service_time_series[0], 2)
+    service_percent_array = [100 - service_percentage, service_percentage]
+    return service_percent_array
+
+def condensed_time_series(total_time_series):
```

Comment:
The way the data was before was [1, 1, 1, 1, 1, 1, 1, 7, 7, 7, 7, 7, 7, 7, 23, 23, 23, 23, 23, 23, 23]; that is, the same number 7 times for each number. I wasn't sure if we wanted to keep it like that since the graph would be extremely long and look like the line card graphs. It seemed like the overall trend of the graph stayed the same and was almost easier to understand without 7 duplications because before the graph was all horizontal lines. Let me know if you want me to change it back though, I wasn't sure.

Reply:
Ahhhh I see. Yes, totally makes sense to remove the duplicates since they hold constant over a week. Could you document this with a comment?

Reply:
yep!